packages/babel-parser/src/tokenizer/types.js

```diff
@@ -139,21 +139,21 @@ export const types: { [name: string]: TokenType } = {
   tilde: new TokenType("~", { beforeExpr, prefix, startsExpr }),
   pipeline: createBinop("|>", 0),
   nullishCoalescing: createBinop("??", 1),
-  logicalOR: createBinop("||", 1),
-  logicalAND: createBinop("&&", 2),
-  bitwiseOR: createBinop("|", 3),
-  bitwiseXOR: createBinop("^", 4),
-  bitwiseAND: createBinop("&", 5),
-  equality: createBinop("==/!=/===/!==", 6),
-  relational: createBinop("</>/<=/>=", 7),
-  bitShift: createBinop("<</>>/>>>", 8),
-  plusMin: new TokenType("+/-", { beforeExpr, binop: 9, prefix, startsExpr }),
-  modulo: createBinop("%", 10),
-  star: createBinop("*", 10),
-  slash: createBinop("/", 10),
+  logicalOR: createBinop("||", 2),
+  logicalAND: createBinop("&&", 3),
+  bitwiseOR: createBinop("|", 4),
+  bitwiseXOR: createBinop("^", 5),
+  bitwiseAND: createBinop("&", 6),
+  equality: createBinop("==/!=/===/!==", 7),
+  relational: createBinop("</>/<=/>=", 8),
```

Comment:
Oh, we'll also need to increase the `in` and `instanceof` operator precedence to match the relational precedence ([spec](https://tc39.es/ecma262/#sec-relational-operators)). That should fix this `typescript/cast/as` test case, which is currently incorrect.

Reply:
For this https://github.com/babel/babel/pull/10269#discussion_r307880391
In the comment you had mentioned to raise the binOp for all other operators in order to lower the binOp for ??.
That is done and after that I faced the cast as error.
That incrementing the binOp for all operators is that correct?